template/README.md

```diff
@@ -1,137 +1,157 @@
-
- Project structure:
- ```text
-|-- main.tf
-|-- variables.tf
-|-- outputs.tf
-|
-|-- example/
-|   |-- main.tf
-|   |-- variables.tf
-|   |-- outputs.tf
-```
+The command, will install:
+- [VirtualBox](https://www.virtualbox.org/)
+- [Packer](https://www.packer.io/)
+- [Vagrant](https://www.vagrantup.com/) with additional plugins
+- [Additional software dependent on the OS (Linux, MacOS)](../install/Makefile)
 
-## Using the template and box
+### Packages that needs to be pre-installed
 
-### Commands
-There are several commands that help to run the vagrant-box:
-- `make install` installs all prerequisites
+- [Make](https://man7.org/linux/man-pages/man1/make.1.html)
+- [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-The-Command-Line)
 
-- `make up` provisions a [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/) box on your machine. After the machine and hashistack are set up it will run the default [startup scheme](#default-startup-scheme).
+#### MacOS Specific
+- Virtualization must be enabled. [This is enabled by default on MacOS.](https://support.apple.com/en-us/HT203296)
+- [Homebrew](https://brew.sh/) must be installed.
 
-- `make clean` take down the provisioned box if there is any.
+#### Ubuntu Specific
+- Virtualization must be enabled. [Error if it is not.](https://github.com/fredrikhgrelland/vagrant-hashistack/issues/136)
+- Packages [gpg](http://manpages.ubuntu.com/manpages/xenial/man1/gpg.1.html) and [apt](http://manpages.ubuntu.com/manpages/bionic/man8/apt.8.html) must be installed.
 
-- `make update` downloads the newest version of the [vagrant-hashistack box](https://github.com/fredrikhgrelland/vagrant-hashistack/) from [vagrantcloud](https://vagrantcloud.com/fredrikhgrelland/hashistack).
+---
 
-- `make example` runs the example in [test_example/](test_example/)
+`NB` _Post installation you might need to reboot your system in order to start the virtual-provider (VirtualBox)_
 
-For full info, check `template/Makefile`
+---
 
-`!Note` - Makefile commands are not idempotent in the context of vagrant-box.  You could face the error of port collisions. Most of the cases it could happen because of the vagrant box has already been running.
 
-Once vagrant-box is running, you can use other options like the Nomad- and Terraform-CLIs to iterate over the deployment in the development stage.
+## Configuration
+
+### Default Startup Scheme
+From a thousand foot view the default startup scheme will:
+1. Start the hashistack and MinIO
+2. Run [playbook.yml](dev/ansible/playbook.yml), which in turn runs all ansible-playbooks inside [dev/ansible/](dev/ansible).
+
+> :bulb: Vagrantfile lines 8-11 run the first playbook on startup, and can be changed.
```

Comment:
It is a bit controversial to me, see the value of this `thousand foot view` description. I would suggest jumping onto `detailed` one here.
```suggestion
```

Reply:
I like it because it is a very rough description, missing lots of nuances, but maybe a little too "casual" of an explanation. Any other opinions @fredrikhgrelland @Neha-Sinha2305?